### PR TITLE
        updates for handling forcing outputs in ladas cycling

### PR DESCRIPTION
**Pull request by saraqzhang**
modified:   src/Applications/GEOSdas_App/GEOSdas.csm
	modified:   src/Applications/GEOSdas_App/fvpsas

### CODE REVIEW

File: src/Applications/GEOSdas_App/GEOSdas.csm [hunk 1]
```diff
@@ -5866,10 +5867,10 @@ endif
 
             # hold lfo files for the next segment of ldas coupling
             #----------------------------------------------------------
-            if ( $LDAS_ANA ) then
-               mkdir -p              $FVHOME/recycle/holdforc
-               /bin/cp  *2d_lfo*nc4  $FVHOME/recycle/holdforc/
-            endif
+            #if ( $LDAS_ANA ) then
```
Comment (gmao-rreichle) [inline src/Applications/GEOSdas_App/GEOSdas.csm]: Is the if condition here intentionally commented out?   Based on a quick look it seems that the if condition should be there, but I might be wrong.

Comment (saraqzhang) [reply src/Applications/GEOSdas_App/GEOSdas.csm]: @gmao-rreichle ,   that block was leftover from before, I forgot to delete it in the update. I will do it now the push the change.  thanks for spotting it.

**Review by gmao-rreichle — DISMISSED** [review]

**Comment by gmao-rreichle** [comment]
Looks ok to me, but I'm not the judge here.
I changed the label to "0 diff trivial" because the edits are all within `if ( $LDAS_ANA )` blocks.
@rtodling, from what I understand, this fix is needed because recent ADAS changes broke the weakly-coupled land assimilation.

**Comment by mathomp4** [comment]
The force push above was:
```
gh repo clone GEOS-ESM/GEOSadas GEOSadas-Reset
cd GEOSadas-Reset
git checkout feature/saraqzhang/ladasupdate
git reset HEAD~5
git add .
git commit -m "fixed typo for zeit_c?.x MovePredictorOutput (fvpsas)"
git push --force origin feature/saraqzhang/ladasupdate
```
The add was for the "fix misspelling" commit that was previously there.

**Comment by saraqzhang** [comment]
@gmao-rreichle @rtodling     
I updated the prompts and defaults concerning LDAS coupling in fvsetup and added the sample input file for ladas setup into testsuites/

**Comment by saraqzhang** [comment]
the changes here are consolidated into PR #275.

**Pull request closed without merging**